runtime/Makefile

```diff
@@ -35,23 +35,23 @@ all: $(CHPL_ENV_HEADER) third-party-pkgs all.helpme
 depend:
 
 clean: clean.helpme
-	rm -f tags TAGS
+	rm -f tags TAGS $(CHPL_ENV_HEADER)
 
 cleanall: cleanall.helpme
-	rm -f tags TAGS
+	rm -f tags TAGS $(CHPL_ENV_HEADER)
 
 cleandeps: cleandeps.helpme
 
 clobber: clobber.helpme
-	rm -f tags TAGS
+	rm -f tags TAGS $(CHPL_ENV_HEADER)
 
 %.helpme:
 	@$(MAKE) -f Makefile.help MAKE_LAUNCHER=0 CHPL_MAKE_RULE=$* $*
 ifneq ($(CHPL_MAKE_LAUNCHER),none)
 	@$(MAKE) -f Makefile.help MAKE_LAUNCHER=1 CHPL_MAKE_RULE=$* $*
 endif
 
-$(CHPL_ENV_HEADER):
+$(CHPL_ENV_HEADER): $(CHPL_MAKE_HOME)/util/printchplenv $(CHPL_MAKE_HOME)/util/chplenv/*
```

Comment:
This dependent list may be overkill.  `util/printchplenv` is effectively just a one-line `bash` wrapper for `util/chplenv/printchplenv.py`.  And `util/chplenv/*` includes not only the `.py` scripts but their corresponding auto-compiled `.pyc` counterparts which are created and updated on the fly.  Maybe just `util/chplenv/*.py util/chplenv/third-party-pkgs` for the dependents?

Reply:
I didn't have any .pyc files in my directory when I wrote this, which is why I didn't think to exclude them, but maybe that's because I'd done a `make clean` recently?  I'm somewhat open to narrowing it down in this way, though it means that this rule would need to be updated if/when other files are added to the directory that don't match those filenames (and I'd rather have the rule cast a wider net than become too conservative again).  Is it the case that .pyc files are only updated when the .py files change?  If so, then the dependence on the .pyc's is overkill but not technically incorrect or overly conservative (in that they're unlikely to result in unneeded re-builds of this header?).

Reply:
I have `.pyc` files in my `util/chplenv` from Monday, but yeah, it looks like they all correspond to `.py` scripts I changed as part of the `chpl_3p_<pkg>configs.py` simplification PR.  So maybe that part of it's okay after all.